internal/stores/objects_test.go

```diff
@@ -185,9 +189,12 @@ func TestSQLObjectStore(t *testing.T) {
 										Host: dbHost{
 											PublicKey: hk2,
 										},
-										FCID:        fcid2,
-										StartHeight: startHeight2,
-										TotalCost:   totalCost2.Big(),
+										FCID:                fcid2,
```

Comment:
can we rename this field to `ID`?

Reply:
Unfortunately there already is an `ÌD` field which gets set by the DB itself in an auto incremented fashion.